File: fakeshop.API/Models/Domain/Product.cs
namespace fakeshop.API.Models.Domain {
    public class Product {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public double Price { get; set; }
        public string ProductImageUrl { get; set; }
        public Guid VendorId { get; set; }

        public Vendor Vendor { get; set; }
    }
}

File: front-end/fakeshop/src/Product.js
function ProductCard({ product }) {
  return (
    <div className="card">
      <img src={product.productImageUrl} alt={product.name} />
      <h3>{product.name}</h3>
      <p>${product.price.toFixed(2)}</p>
      <p>{product.vendor?.name}</p>
      <button>Add to Cart</button>
    </div>
  );
}

export default ProductCard;